src/main/java/io/ark/core/requests/AccountManager.java

```diff
@@ -1,31 +1,25 @@
 package io.ark.core.requests;
 
-import static io.ark.core.requests.AccessType.ARRAY;
-import static io.ark.core.requests.AccessType.FLOAT;
-import static io.ark.core.requests.AccessType.OBJECT;
 import static io.ark.core.util.Constants.ARKTOSHI;
 
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
 import org.bitcoinj.core.ECKey;
 import org.bitcoinj.crypto.MnemonicCode;
 import org.bitcoinj.crypto.MnemonicException;
-import org.json.JSONArray;
 
 import io.ark.core.config.Options;
 import io.ark.core.crypto.Crypto;
-import io.ark.core.exception.ArkNodeException;
 import io.ark.core.model.Account;
 import io.ark.core.model.Delegate;
 import io.ark.core.network.NetworkConfig;
 import io.ark.core.network.NetworkInfo;
-import io.ark.core.util.JsonUtils;
+import io.ark.core.responses.AccountResponse;
 
 public class AccountManager extends Manager {
 
-  private static final String createAccount = "/api/accounts?address=";
+  private static final String getAccount = "/api/accounts?address=";
   private static final String getDelegateFee = "/api/accounts/delegates/fee";
   private static final String getDelegates = "/api/accounts/delegates?address=";
```

Comment:
I'm considering putting all endpoints in a properties file since there's a chance they will change and it'd be nice to be able to update without recompiling the module.